src/cgame/cg_rocket_datasource.cpp

```diff
@@ -34,6 +34,47 @@ Maryland 20850 USA.
 
 #include "cg_local.h"
 
+static void StripNonPrintingCharacters( std::string& str )
+{
+	str.erase(
+		std::remove_if(
+			str.begin(),
+			str.end(),
+			[]( unsigned char c ){
+				return !std::isprint( c );
+			}),
+		str.end() );
+}
+
+static void StripSpacingCharacters( std::string& str )
+{
+	str.erase(
+		std::remove_if(
+			str.begin(),
+			str.end(),
+			[]( unsigned char c ){
+				return std::isspace( c );
```

Comment:
Don't use std::isspace because the behavior is not specified in different locales. Use Str::cisspace

Reply:
Done.